platform/core/src/services/DicomMetadataStore/createStudyMetadata.js

```diff
@@ -49,7 +49,12 @@ function createStudyMetadata(StudyInstanceUID) {
       );
 
       if (existingSeries) {
-        existingSeries.instances.push(...instances);
+        const sopMap = {};
```

Comment:
The DicomMetadataStore was growing on each subsequent view of the same study, because the instances were being added regardless.

Reply:
can you add this comment in the code please? we shouldn't use github for commenting/explaining code if it is required explanation. Something like

`Filter ....`

Reply:
Done